docs/modding/README.md

```diff
@@ -1,5 +1,6 @@
-# Modding
+# [Titanfall 2 and Northstar Modding](https://r2northstar.readthedocs.io/)
 
-The documentation for modding has been moved over to [r2northstar.readthedocs.io](https://r2northstar.readthedocs.io)
+### Modding documentation for Northstar and Titanfall 2 has moved over to: https://r2northstar.readthedocs.io/.
 
-For skin creation, see [https://titanfall-skin-group.gitbook.io/titanfall-2-skin-creation/](https://titanfall-skin-group.gitbook.io/titanfall-2-skin-creation/)
\ No newline at end of file
+# [Custom Skin Modding](https://retrywiki.gitbook.io/tf2/)
+### RETRY Titanfall 2 Custom Skin & Textures Wiki & Guides Gitbook is at: https://retrywiki.gitbook.io/tf2/.
```

Comment:
E3VL has transferred the responsibility of managing the skins wiki to SenorGeese. 
The new wiki just hasn't been shown off as much within the Northstar discord.

Reply:
Yup. You can contact E3VL for verification. 
It's a different wiki though since transferring ownership would cost money (to re-activate the organization) and so I could properly structure the wiki to the Github repo and separate the twos wiki's.